src/web/arr/trove/chart.arr

```diff
@@ -1873,6 +1945,38 @@ fun stacked-bar-chart-from-list(
   data-series.make-axis(max-positive-height, max-negative-height)
 end
 
+fun residual-chart-from-list(
```

Comment:
Can we rename this `interval-chart-from list`, in keeping with the data series renaming?